plugins/csmodel.js

```diff
@@ -591,6 +591,10 @@ Plugin.register('csmodel', {
 					} else {
 						codec.parse(files[0].content);
 					}
+					csname = files[0].name.replace(".csmodel","").replace(" ","_").toLowerCase();
```

Comment:
Make sure to use regex to replace the white spaces with underscores. It's a little quirk in Javascript that if you replace a string, it will only replace the first occurrence.
e. g. `.replace(/\s+/, '_')`

Reply:
Gotcha and thanks for the pointer, I'll correct that.